sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -464,16 +545,92 @@ public async Task StartRecognizeCustomFormsWithLabelsCanParseMultipageForms(bool
 
             var recognizedForm = recognizedForms.Single();
 
-            ValidateRecognizedForm(recognizedForm, includeTextContent,
+            ValidateRecognizedForm(recognizedForm, includeTextContent: true,
                 expectedFirstPageNumber: 1, expectedLastPageNumber: 2);
 
-            for (int formIndex = 0; formIndex < recognizedForms.Count; formIndex++)
+            // Add fields assertions when https://github.com/Azure/azure-sdk-for-net/issues/12139
+            // is solved.
+
+            // Basic sanity test to make sure pages are ordered correctly.
+
+            for (int pageIndex = 0; pageIndex < recognizedForm.Pages.Count; pageIndex++)
             {
-                // Basic sanity test to make sure pages are ordered correctly.
-                // TODO: implement sanity check once #11821 is solved.
+                var formPage = recognizedForm.Pages[pageIndex];
+                var sampleLine = formPage.Lines[3];
+                var expectedText = pageIndex == 0 ? "Bilbo Baggins" : "Frodo Baggins";
+
+                Assert.AreEqual(expectedText, sampleLine.Text);
             }
         }
 
+        [Test]
+        public async Task StartRecognizeCustomFormsWithLabelsCanParseBlankPage()
```

Comment:
Currently there are 4 operations we are testing: content recognition, receipts recognition, custom form recognition without labels and custom form recognition with labels. For each one of them we have a "multipage form with a blank page" test.

The custom form recognition with labels, however, behaves differently from others because it treats the whole file as a single multipage form, while other operations parse each page individually. In order to test an actual "blank form" scenario, I've added a special case which makes use of the `blank.pdf` file, a single blank page. Just wanted to make sure that having a complete blank form in the custom form with labels scenario wouldn't break our client with unexpected responses from the service.